tox.ini

```diff
@@ -211,10 +217,9 @@ commands_pre =
 
   grpc: pip install {toxinidir}/ext/opentelemetry-ext-grpc[test]
 
-  wsgi,flask,django,asgi,pyramid: pip install {toxinidir}/tests/util
+  wsgi,flask,django,asgi,pyrmaid,starlette: pip install {toxinidir}/tests/util
```

Comment:
This might be a dumb question, but how come the env names specified in `commands_pre` are different than the ones in `envlist` and `change_dir`? e.g. test-ext-redis vs redis

Reply:
there is no behavioral difference. Tox tokenizes environment names, delimitting by dashes:

https://tox.readthedocs.io/en/latest/config.html#tox-environments

Which I figured out after working around the "test-instrumentation" matching. There's an argument we could simplify all of this, but I don't have a concrete sketch.